l10n_it_pos_fatturapa/static/src/js/screens.js

```diff
@@ -0,0 +1,53 @@
+odoo.define('l10n_it_pos_fatturapa.screens', function (require) {
+    "use strict";
+
+    var Screens = require('point_of_sale.screens');
+
+    Screens.ClientListScreenWidget.include({
+        _update_client_name: function(checked){
+            if (!checked) {
+                var name = ($('.lastname').val() || '') + ' ' + ($('.firstname').val() || '');
```

Comment:
Teoricamente l'ordine è stabilito dal parametro `partner_names_order` di `partner_firstname`.
Altrimenti se l'utente imposta ad esempio l'ordine `nome cognome` poi qua il sistema  lo inverte.

Però più andiamo avanti più mi convinco che bisognerebbe mettere tutta questa roba in un modulo `pos_partner_firstname` su https://github.com/OCA/pos/

@robyf70 che ne pensi?

Reply:
@eLBati ... penso che se facciamo il ``pos_partner_firstname`` dovremmo solo mettere la funzione di composizione corretta del ``name``, ma poi ``firstname`` e ``lastname`` dovrebbero essere messi sempre in questo repo oppure vanno da un'altra parte?

Reply:
`partner_firstname` introduce `firstname` e `lastname`, quindi il modulo `pos_partner_firstname` si occuperebbe di gestire questi campi sul POS